src/libstd/primitive_docs.rs

```diff
@@ -255,6 +255,11 @@ mod prim_unit { }
 ///
 /// *[See also the `std::ptr` module](ptr/index.html).*
 ///
+/// [`null`]: ../std/ptr/fn.null.html
```

Comment:
Too many PRs of fine so I can't find it but it was common when we linked from core to not find the equivalent in core so I used to always target std. Maybe things have changed since then?